doc/source/docs/howto_build_docs.rst

```diff
@@ -57,7 +57,9 @@ To build the PDF documentation, do instead::
    make latex
    make -C build/latex all-pdf
 
-You will need to have Latex installed for this.
+You will need to have Latex installed for this, inclusive of support for
+Greek letters.  For example, on Ubuntu xenial ``texlive-lang-greek`` and
+``cm-super`` are needed.  Also ``latexmk`` is needed on non-Windows systems.
```

Comment:
Currently, the `texlive-lang-greek` and  `cm-super` dependencies could be avoided via this diff
```diff
diff --git a/numpy/doc/constants.py b/numpy/doc/constants.py
index 72793e44d..fc3ca4b2c 100644
--- a/numpy/doc/constants.py
+++ b/numpy/doc/constants.py
@@ -51,7 +51,7 @@ add_newdoc('numpy', 'e',
 
 add_newdoc('numpy', 'euler_gamma',
     """
-    ``γ = 0.5772156649015328606065120900824024310421...``
+    :math:`\\gamma` ``= 0.5772156649015328606065120900824024310421...``
 
     References
     ----------
```
and then one *must* (else SPhinx 2.x mechanism for Greek is triggered) remove `LGR` from `latex_elements['fontenc']` configuration in `doc/source/conf.py`.

Reply:
We should document what we need to do, so either use gamma and remove `LGR` or leave it as-is.